Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -38,6 +38,15 @@
     <string name="wiimote_tilt">Tilt</string>
     <string name="wiimote_shake">Shake</string>
 
+    <!-- WARNING Do not move these controller entries AT ALL COSTS! They are indexed with ints, and an assumption
+            is made that they are placed together so that we can access them sequentially in a loop.
+            Wiimotes start at 4 since they are mapped to padID's 4-7 in the native code.-->
```

Comment:
Accidentally included?

Reply:
Should I remove this for the Wii Remote Extensions? There were no string entries for them.